openhtf/plugs/__init__.py

```diff
@@ -142,6 +143,15 @@ class BasePlug(object):
   # Allow explicitly disabling remote access to specific attributes.
   disable_remote_attrs = set()
 
+  class Placeholder(object):
```

Comment:
placeholder is a confusing name, but I'm at a loss for better ones :/

However, `self.cls` is really not helping in the naming department. Consider `self.base_class` or similar

Lastly, this can use the new [util.classproperty][0] to become a better API and potentially remove this class altogether

[0]: https://github.com/google/openhtf/blob/master/openhtf/util/__init__.py#L95